FULL_HELP_DOCS.md

```diff
@@ -163,6 +163,7 @@ Utilities to manage contract aliases
 * `remove` — Remove contract alias
 * `add` — Add contract alias
 * `show` — Show the contract id associated with a given alias
+* `list` — List all stored aliases for a given network
```

Comment:
Should it be `ls` instead? Also maybe `remove` -> `rm`?
To use the same style as `stellar keys`/`stellar network`

Reply:
Good catch. I think so. I'll rename it in a separate pr.